steps/common/steps.go

```diff
@@ -56,6 +59,19 @@ func (cs Steps) InitializeSteps(ctx context.Context, scenCtx *godog.ScenarioCont
 		}
 		return fmt.Errorf("mismatch of values: expected '%s', actual '%s'", e, v)
 	})
+	scenCtx.Step(`^I store own ip in context "([^"]*)"$`, func(key string) error {
+		log := logrus.WithField("scenario", "ctx.Name")
+		log.Debug("saving ip in context")
```

Comment:
We're not using these logs invokation inside Golium steps.go files.  We could remove them.

Reply:
You are right, thanks!!